platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -210,6 +220,11 @@ private MigrationReportImpl doExport(
       } else {
         SecurityLogger.audit("Failed to delete exported file {}", exportFile);
       }
+      if (FileUtils.deleteQuietly(keyFile.toFile())) {
```

Comment:
❓ Any reason you're not using your private `deleteQuietly` method for these deletions-with-audits?

Reply:
@coyotesqrl I think it is just simpler since it handles all errors and return false.
❓ That being said, do we need to audit the deletion
❗️ We should make sure to delete all 3 files in case of failures and not just one.

Reply:
I updated it so that each of the files is now deleted using the private method